draft-ietf-quic-qpack.md

```diff
@@ -249,18 +250,19 @@ Because QUIC does not guarantee order between data on different streams, a
 header block might reference an entry in the dynamic table that has not yet been
 received.
 
-Each header block contains a Largest Reference ({{header-prefix}}) which
-identifies the table state necessary for decoding. If the greatest absolute
-index in the dynamic table is less than the value of the Largest Reference, the
-stream is considered "blocked."  While blocked, header field data SHOULD remain
-in the blocked stream's flow control window.  When the Largest Reference is
-zero, the frame contains no references to the dynamic table and can always be
-processed immediately. A stream becomes unblocked when the greatest absolute
-index in the dynamic table becomes greater than or equal to the Largest
-Reference for all header blocks the decoder has started reading from the stream.
-If the decoder encounters a header block where the actual largest reference is
-not equal to the Largest Reference declared in the prefix, it MAY treat this as
-a stream error of type HTTP_QPACK_DECOMPRESSION_FAILED.
+Each header block contains the total number of entries added to the dynamic
+table - the Insert Count - which identifies the table state necessary for
+decoding (see {{header-prefix}}). If the Insert Count is greater than the number
```

Comment:
I think this will be somewhat confusing, because the state of the table at the encoder at the time of encoding (insert count) can be different than the table state (insert count) required to decode a particular block.  Maybe if you called it 'Required Insert Count' in the header prefix that would make it clear that an encoder shouldn't just blindly use the actual Insert Count?